adapters/taboola/taboola.go

```diff
@@ -35,9 +39,17 @@ func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapte
 		return nil, []error{err}
 	}
 
+	requestEndpoint := a.endpoint
+	if len(request.Imp) > 0 {
```

Comment:
There is no need to check this. Request always has at least one impression.

Reply:
check removed